### ISSUE TITLE
Section VIII : A Varation of a Convolutional encoder-decoder architecture for semantic segmentation

### ISSUE THREAD

**Issue by nikhilrayaprolu** [AIcrowd/mapping-challenge-review#5]
- [ ]  The proposed method in the paper is essentially the U-Net. Why is it branded differently ? What are the key differences (if at all) ?
    - [ ] Table IV reports that Net12 performance better on the validation set. But Test set results there are not provided.
        The whole table should be arranged to consistently report losses on Test Set and Validation set for all variants possible
        For ex all possible combinations : {Net6 , Net8 , Net 10, Net 12} x { Morphology Di, Morphology Er } x {Test Set, Validation Set} x {Average Precision} x {Average Recall}

**Comment by fleer** [comment]
We changed the title to "Comparing an adapted U-Net Architecture for Varying Depths" and adjusted the beginning of our section:

In this section, we are analysing and comparing a U-Net like structure \cite{unet} for different depths. The architecture was derived originally from a convolutional AutoEncoder structure as used for reconstructing images (see, for an example the keras tutorial \cite{chollet2015keras}). This AutoEncoder-type architecture was modified for semantic segmentation: the provided ground truth annotations of the buildings were used as targets for training in order to accomplish the detection of buildings. Furthermore, skip-connections were introduced as found in U-net that connect encoding and decoding blocks on the same level. These connections help to recover spatial information and in our experiments this provided better reconstruction of details in images compared to post-processing, for example, using conditional random fields \cite{kraehenbuehl2012}. This architecture differs from U-net, first, with respect to the sequence inside the decoding blocks. Following the AutoEncoder approach, the decoder block mimics exactly the encoder block and consists of a single convolution followed by upsampling (max-pooling is used in the encoder block). Second, we used a single convolution of size $5 \times 5$. This architecture was used for different depth (stacked blocks of encoders and decoders).
